packages/plugins/rubygem-foreman_salt/rubygem-foreman_salt.spec

```diff
@@ -4,12 +4,12 @@
 
 %global gem_name foreman_salt
 %global plugin_name salt
-%global foreman_min_version 1.24.0
+%global foreman_min_version 2.5.2
```

Comment:
This is a very specific number. Why this version instead of just 2.5?

Reply:
I just set it to the most up-to-date version when I started the PR, since the Autosign procedure (which is part of the new Foreman Salt version) needed [this](https://github.com/theforeman/foreman/commit/ce2190057571ad3d9c3db3f6a80b9dcf52448426) commit to be part of Foreman in order to work properly. I wanted to update the `foreman_min_version` when the corresponding PR got merged.

It has been merged by now, but there isn't a new Foreman release which includes it. So, I guess we have to wait until 3.2 is released, right?

Reply:
Alright, thanks! I set it to 2.5 now which is the version I tested it for initially.